CRM/Contribute/BAO/Contribution.php

```diff
@@ -3376,12 +3376,10 @@ public static function isSubscriptionCancelled($contributionId) {
    *   Contribution object, line item array and params for trxn.
    *
    *
-   * @param array $financialTrxnValues
-   *
    * @return null|\CRM_Core_BAO_FinancialTrxn
    */
-  public static function recordFinancialAccounts(&$params, $financialTrxnValues = NULL) {
-    $skipRecords = $update = $return = FALSE;
```

Comment:
In practice it makes no difference since it's obviously not serving a purpose but wondering if maybe $update was a typo intended to be $updated, which is a var used later.

Reply:
One of those mysteries we may never know the answer to :-)